csatar-plugins/csatar/updates/builder_table_update_csatar_csatar_scouts_3.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Csatar\Csatar\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableUpdateCsatarCsatarScouts3 extends Migration
+{
+    public function up()
+    {
+        Schema::table('csatar_csatar_scouts', function($table)
+        {
+            $table->integer('legal_relationship')->unsigned();
```

Comment:
This column is not needed. Please remove it with a new migration and don't edit old migration files.